ctapipe_io_magic/laser_params_corr_test.py

```diff
@@ -0,0 +1,411 @@
+import logging
+import datetime
+from pathlib import Path
+import os
+import numpy as np
+import astropy.units as u
+from ctapipe_io_magic import MAGICEventSource
+
+logger = logging.getLogger(__name__)
+logger.addHandler(logging.StreamHandler())
+logger.setLevel(logging.INFO)
+
+class ReportLaser:
+    def magic_reports_reading(self, input_file, process_run=False):
+        logger.info(f"\nInput file: {input_file}")
+        event_source = MAGICEventSource(input_file, process_run=process_run)
+        is_simulation = event_source.is_simulation
+        logger.info(f"\nIs simulation: {is_simulation}")
+
+        obs_id = event_source.obs_ids[0]
+        tel_id = event_source.telescope
+
+        logger.info(f"\nObservation ID: {obs_id}")
+        logger.info(f"Telescope ID: {tel_id}")
+
+        laser = event_source.laser
+
+        for key, report_list in laser.items():
```

Comment:
this does not fit the current scheme of writing the reports: list rather then a dictionary
and the whole loop is not needed because it just shown a debug message